src/modules/map/i18n/contextMenu.provider.js

```diff
@@ -12,7 +12,9 @@ export const provide = (lang) => {
 				map_contextMenuContent_copy_icon: 'Copy to clipboard',
 				map_contextMenuContent_clipboard_error: '"Copy to clipboard" is not available',
 				map_contextMenuContent_clipboard_success: 'was copied to clipboard',
-				map_contextMenuContent_clipboard_link_text: 'The link'
+				map_contextMenuContent_clipboard_link_text: 'The link',
```

Comment:
must be added to `/opt/jsDev/bav4-review/src/modules/toolbox/i18n/toolbox.provider.js`
and registered under `toolbox_shareToollDialog_...`